vipps-recurring-api.md

```diff
@@ -752,83 +775,58 @@ to set up a new agreement.
-`PROCESSING` to `CHARGED`, or even from `PROCESSING` to `REFUNDED`
-depending on your systems. `PROCESSING` is not normally seen, as a charge usually
-only has this status for a very brief period of time, but merchants must
-still make sure they can handle this status.
-
-### Charge failure reasons
-
-When fetching a charge through the API, you can find two fields in the response
-body to identify why the charge failed `failureReason` and `failureDescription`
-
-**Important:** This is an experimental feature. The design has not been finalized
-and it only applies to Vipps Recurring API. You may use `failureReason` and
-`failureDescription` when the fields are available in the response, but do
-not depend on them, and be aware that they may only contain `None` if there
-are no further details available for the failure.
-
-An example from a response:
-
-```json
-{
-  "status": "FAILED",
-  "type": "RECURRING",
-  "failureReason": "insufficient_funds",
-  "failureDescription": "Payment was declined by the payer bank due to lack of funds"
-}
-```
-
-Here is a list of possible values for `failureReason`, their respective descriptions and possible actions that the user/merchant could take.
-
-| Reason | Description | Action |
-| ---- | ----------- | ------ |
-| insufficient_funds | Payment was declined by the payer bank due to lack of funds. | User must either add funds to the card to cover the difference between the amount to be paid. Alternatively they can change to another, or add a new, payment source that is adequately funded to complete the transaction. |
-| invalid_card | The user tried to pay using a card that has either expired or is disabled by the issuer. | User must change, or add a new, payment source on the agreement in Vipps. |
-| verification_required | Payment declined because the issuing bank requires verification. | Ask the user to change, or add a new, payment source on their agreement in Vipps. Alternatively removing and then adding the card might solve the issue. |
-| invalid_payment_source | The provided payment source is disabled or does not exist. | User must change payment source for the agreement. |
-| charge_amount_too_high | Amount is higher than the users specified max amount | The user have a lower `maxAmount` on the variableAmount agreement than the amount of the charge. The user must update their `maxAmount` on the agreement for the charge to be processed.
-| internal_error | Internal Error / Something went wrong | The error could not be identified as one of the above. Try to create the charge again, changing or adding payment sources on the agreement, or contact Vipps for more information. |
+`PENDING` to `CHARGED`, or even from `PENDING` to `REFUNDED`
+depending on your systems.
+The `DUE` status is not normally seen, as a charge usually only has this status
+for a brief period of time, but merchants must still make sure they can handle
+this, and all other, statuses.
```

Comment:
We will probably not change things so it will be like this. and DUE wil certainly NOT be a status that is normally not seen. It is the primary status a charge has for up to 30 days before the due date.

Reply:
Yes, sorry, thanks, fixed